tests/test_servicenow.py

```diff
@@ -0,0 +1,273 @@
+import os
+import sys
+import logging
+import requests
+
+from unittest import main, TestCase
+from unittest.mock import patch
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../ticketutil/'))
```

Comment:
It's still less messy than the alternative. You can see this line in E2E-Container unit tests. Without it there are problems when patch calls servicenow module.